resource/plugins/unreal/python/publisher/collectors/widget/unreal_dependencies_publisher_collector_options.py

```diff
@@ -0,0 +1,55 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2022 ftrack
+
+from ftrack_connect_pipeline_unreal import plugin
+from ftrack_connect_pipeline_qt.plugin.widget.base_collector_widget import (
+    BaseCollectorWidget,
+)
+
+import ftrack_api
+
+
+class UnrealDependenciesPublisherCollectorOptionsWidget(BaseCollectorWidget):
```

Comment:
Are we still in need of this widget?

Reply:
Yes, it is used to visualise the dependencies during publish. Artist migh also want to add dependencies manually here in the future.